src/main/java/seedu/address/logic/commands/DeleteAnimalCommand.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.AnimalModel;
+import seedu.address.model.animal.Animal;
+
+/**
+ * Deletes a person identified using it's displayed index from the address book.
+ */
+public class DeleteAnimalCommand extends AnimalCommand {
+
+    public static final String COMMAND_WORD = "delete";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the animal identified by the index number used in the displayed animal list.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_DELETE_ANIMAL_SUCCESS = "Deleted Animal: %1$s";
```

Comment:
I am not sure if the use of `1$` is required, since this format string only takes in 1 argument. I would suggest changing it to just `%s` for improved readability.